analytics/Makefile

```diff
@@ -146,6 +146,17 @@ sprint-data-export:
 	--project $(SPRINT_PROJECT) \
 	--output-file $(SPRINT_FILE)
 
+# Import file data to database
+# Takes sprint and issue and puts it in database
+sprint-db-data-import:
+	@echo "=> Importing project data to the database"
+	@echo "====================================================="
+	$(POETRY) analytics export db_export \
```

Comment:
⭐⭐⭐  &nbsp; _(must change before approval)_ The makefile command says db-**import** whereas the python command is db-**export**. You'll want to pick one or the other. I'm thinking import?

Reply:
I'm also thinking `import`

Reply:
Please update with import. Please also create a separate import app: 

```python
import_app = typer.Typer()
...
app.add_typer(import_app, name="import", help="Import data into the database")

```